cmd/build.go

```diff
@@ -534,6 +535,10 @@ func updateDeploymentConfig(config *buildCommandConfig, imageName string, labels
 
 	appsodyApplication.Spec.ApplicationImage = imageName
 
+	if appsodyApplication.Namespace == "" {
```

Comment:
Should we always overwrite the namespace from the config? I'm not sure of the answer here, but it seems odd to ignore the `config.namespace` if it was set.

Reply:
In the case where a namespace is specified in the config file and a namespace is passed as an argument, we check if they are the same and if not we through an error. This means that it only make sense to write to the config file if the namespace field is empty, if not we will just be writting the same thing.

Reply:
Ok, I think that makes sense, but it is not clear in the code. Would you please put a code comment here explaining this and its dependency with deploy?